org.jcryptool.core.introduction/src/org/jcryptool/core/introduction/views/AlgorithmInstruction.java

```diff
@@ -260,27 +264,30 @@ public void createPartControl(Composite parent) {
 		gl_content.horizontalSpacing = 0;
 		gl_content.verticalSpacing = 0;
 		content.setLayout(gl_content);
+		content.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
 //		content.setBackground(ColorService.GREEN);
 		
 		scrolledComposite.setContent(content);
 
 
 		initializeScaledImages();
 		
-		
-		// Composite that should grab space on the left.
-		// This should center the slideshow
-		Composite left = new Composite(content, SWT.NONE);
-		GridData gd_left = new GridData(SWT.FILL, SWT.FILL, false, false);
-		left.setLayoutData(gd_left);
-		GridLayout gl_left = new GridLayout();
-		gl_left.marginHeight = 0;
-		gl_left.marginWidth = 0;
-		left.setLayout(gl_left);
```

Comment:
left and right-padding to center something was only a last-ditch effort when something does not play nice with the layout. Is fixed and not needed any more.